README.md

```diff
@@ -13,6 +13,16 @@ npm install --save redux-actions
 import { createAction, handleAction, handleActions } from 'redux-actions';
 ```
 
+### Note
+
+We are also supporting a UMD build:
```

Comment:
Can you please add the npmcdn link here?

`We are also supporting a [UMD build](https://npmcdn.com/redux-actions/dist/redux-actions.min.js)`